### PR TITLE
[FIX] Report for Linear Regression #1079

### PR DESCRIPTION
**Pull request by kumargaurav2596**
The function send_report(self)  in Orange/widgets/regression/owlinearregression.py has been updated to generated report as requested in the issue Report for Linear Regression #1079

### CODE REVIEW

File: Orange/widgets/regression/owlinearregression.py [hunk 1]
```diff
@@ -171,7 +171,17 @@ def commit(self):
...
     def send_report(self):
-        pass
+        self.report_items("Learner Name",[("Learner Name", self.learner_name)])
+
+        regression_type = ["No regularization", "Ridge regression (L2)", "Lasso regression (L1)", "Elastic net regression"]
+
+        parameters =[("Ridge", self.ridge),
+             ("Regression Type", regression_type[self.reg_type]),
+             ("Alpha", self.alpha_index),
+             ("L1 Ratio", self.l1_ratio)]
```
Comment (kernc) [inline Orange/widgets/regression/owlinearregression.py]: Please align these with the open brace.

File: Orange/widgets/regression/owlinearregression.py [hunk 2]
```diff
@@ -171,7 +171,17 @@ def commit(self):
         self.send("Coefficients", coef_table)
 
     def send_report(self):
-        pass
+        self.report_items("Learner Name",[("Learner Name", self.learner_name)])
+
+        regression_type = ["No regularization", "Ridge regression (L2)", "Lasso regression (L1)", "Elastic net regression"]
```
Comment (kernc) [inline Orange/widgets/regression/owlinearregression.py]: Please list these one per line.

File: Orange/widgets/regression/owlinearregression.py [hunk 3]
```diff
@@ -171,7 +171,17 @@ def commit(self):
...
+
+        parameters =[("Ridge", self.ridge),
+             ("Regression Type", regression_type[self.reg_type]),
+             ("Alpha", self.alpha_index),
+             ("L1 Ratio", self.l1_ratio)]
+        self.report_items("Model Parameters", parameters)
+
```
Comment (kernc) [inline Orange/widgets/regression/owlinearregression.py]: These empty lines – please vstrip them.

**Comment by kumargaurav2596** [comment]
I have tidied up the code. Please the merge the PR if rest of it is alright.

**Comment by kernc** [comment]
I'm so sorry, but this has already been resolved in https://github.com/biolab/orange3/pull/1092. Your changeset looks almost acceptable, though. :+1:

**Pull request closed without merging**